### PR TITLE
Сброс пароля

### PR DESCRIPTION
**Pull request by NikitaGoroshko7**
https://trello.com/c/FifEn3ta/14-%D1%81%D0%B1%D1%80%D0%BE%D1%81-%D0%BF%D0%B0%D1%80%D0%BE%D0%BB%D1%8F
Has been added actions ForgotPassword and ResetPassword
Has been added models for validation for actions
Has been added Views for actions
Has been added folder for Services and has been added EmailService for send email
Has been added nuget package for send email MailKit
Has been added style for Views
Has been added global usings for works with EmailService and for work Service

### CODE REVIEW

File: CrowdFundingApplication/Services/EmailService.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        emailMessage.Subject = subject;
+        emailMessage.Body = new TextPart(MimeKit.Text.TextFormat.Html)
+        {
+            Text = message
+        };
+
+        using (var client = new MailKit.Net.Smtp.SmtpClient())
+        {
+            await client.ConnectAsync("smtp.gmail.com", 587, false);
+            await client.AuthenticateAsync("viewstarsettings@gmail.com", "899adminviewstar");
```
Comment (GalayM) [inline CrowdFundingApplication/Services/EmailService.cs]: You can’t do this, the secret data must be put in the config and not committed to GIT, but set through environment variables or read from the secret storage.

File: CrowdFundingApplication/wwwroot/css/ForgotAndResetPage.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
+﻿@import url('https://fonts.googleapis.com/css2?family=Signika+Negative:wght@400;500;600&display=swap');
+
+body {
+    font-family: "Signika", sans-serif;
+    font-weight: 500;
+    letter-spacing: 1px;
+}
+
+#main {
```
Comment (GalayM) [inline CrowdFundingApplication/wwwroot/css/ForgotAndResetPage.css]: Better to bind to a class

Comment (NikitaGoroshko7) [reply CrowdFundingApplication/wwwroot/css/ForgotAndResetPage.css]: Changed

File: CrowdFundingApplication/Services/EmailService.cs (hunk 1 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/Services/EmailService.cs]: Changed

**Review by GalayM — APPROVED** [review]

**Pull request merged**